tpu-client-next/src/quic_networking.rs

```diff
@@ -32,7 +35,10 @@ pub(crate) fn create_client_config(client_certificate: &QuicClientCertificate) -
     let transport_config = {
         let mut res = TransportConfig::default();
 
-        let timeout = IdleTimeout::try_from(QUIC_MAX_TIMEOUT).unwrap();
+        let timeout = IdleTimeout::try_from(Duration::from_secs(
```

Comment:
This should not be changed to 2 seconds as it will break tpu-client-next!

Reply:
Right, I having other batches of changes coming.

Reply:
Reverted this change